mlflow/server/js/package.json

```diff
@@ -16,7 +16,10 @@
     "prettier:fix": "prettier . --write",
     "prettier:check": "prettier . --check",
     "i18n": "node scripts/extract-i18n.js",
-    "i18n:check": "yarn i18n --lint"
+    "i18n:check": "yarn i18n --lint",
+    "graphql-codegen": "cd ../../.. && python ./dev/proto_to_graphql/code_generator.py && cd mlflow/server/js && yarn graphql-codegen:clean && yarn graphql-codegen:base",
```

Comment:
should we run this in CI to ensure `graphql-codegen.ts` is not outdated?

Reply:
@harupy +1 to running corresponding lint script (which will run codegen and assert no changes)

Reply:
sounds good let's do this